cost-analyzer/charts/prometheus/templates/server-deployment.yaml

```diff
@@ -204,7 +204,7 @@ spec:
       {{- range .Values.server.extraSecretMounts }}
         - name: {{ .name }}
           secret:
-            secretName: {{ .secretName }}
+            secretName: {{ tpl .secretName $ }}
```

Comment:
This allows us to use a template in the `values-agent.yaml` (if `.secretName` is just a regular string, it just uses the string directly, so this only enhances the processing of the values input).